docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -27,67 +27,66 @@ Scientists from across multiple labs and organizations have volunteered to revie
+    |                  | Gillian Petro (@gspetro-NOAA)                  | Documentation                                                                     |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Edward Snyder (@EdwardSnyder-NOAA)             | WE2E testing, input data                                                          |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
+    | GLERL            | David Wright (@dmwright526)                    | FVCOM integration, output visualization, rreprocessing tasks                      |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
+    | GSL              | Jeff Beck (@JeffBeck-NOAA)                     | SRW App configuration/workflow, code management, meteorological evaluation        |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Gerard Ketefian (@gsketefian)                  | ``regional_workflow`` scripts, jinja templates, and verification tasks            |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Linlin Pan (@panll)                            | Workflow, CCPP/physics, verification                                              |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Christina Holt (@christinaholtNOAA)            | Workflow, conda environment support, and testing                                  |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Christopher Harrop (@christopherwharrop-noaa)  | Rocoto                                                                            |
+    |                  +------------------------------------------------+-----------------------------------------------------------------------------------+
+    |                  | Daniel Abdi (@danielabdi-noaa)                 | Workflow generation, testing RRFS on the cloud, environment modules               |
+    +------------------+------------------------------------------------+-----------------------------------------------------------------------------------+
```

Comment:
You can add GSL for the above five people.

Reply:
On [RTD](https://srw-ug.readthedocs.io/en/feature-templates/ContributorsGuide.html), GSL actually applies to all 5 of them (and NCAR to Will), tho I can see how that might be confusing looking through the .rst files without looking at RTD. Do you want me to change it to list organization on every line?

Reply:
Oh!  Sorry, I didn't realize things look just fine in RTD.  No need to make any modifications, as it looks good!